### ISSUE TITLE
Cant login after importing old DB

### ISSUE THREAD

**Issue by whiteyhat** [chatwoot/chatwoot#700]
## Information
I imported my previous DB and now I can not actually log in. It prints the notification: 
```
Could not connect to Woot Server, Please try again later
```

If I put a wrong password it detects the password does not match.

I have already tried: 
```
bundle exec rails db:reset
bundle exec rails db:setup
```
And it works again. But I wanna use my backup database...

I have also tried, but it did not work:
```
bundle exec rails db:schema:cache:clear
bundle exec rails db:schema:cache:dump
```

## Version
v1.2.2

**Comment by sojan-official** [comment]
could you run `bundle exec rails db:migrate` after importing the old database. if the issue is persisting can we take a look at the application server logs?

**Comment by tabrza** [comment]
If you're on heroku, would be worth double checking the redis is provisioned

**Comment by whiteyhat** [comment]
> could you run `bundle exec rails db:migrate` after importing the old database. if the issue is persisting can we take a look at the application server logs?

@sojan-official  It did not make a difference... There are literally no error logs... 
```bash
chat_app.1.x6zhcejkwzz4@satoshis-games    | + rm -rf /app/tmp/pids/server.pid
chat_app.1.x6zhcejkwzz4@satoshis-games    | + rm -rf /app/tmp/cache/assets /app/tmp/cache/bootsnap-compile-cache /app/tmp/cache/bootsnap-load-path-cache /app/tmp/cache/webpacker
chat_app.1.x6zhcejkwzz4@satoshis-games    | Waiting for postgres to become ready....
chat_app.1.x6zhcejkwzz4@satoshis-games    | + echo 'Waiting for postgres to become ready....'
chat_app.1.x6zhcejkwzz4@satoshis-games    | + PGPASSWORD=test
chat_app.1.x6zhcejkwzz4@satoshis-games    | + PSQL='pg_isready -h postgres -p 5432 -U postgres'
chat_app.1.x6zhcejkwzz4@satoshis-games    | + pg_isready -h postgres -p 5432 -U postgres
chat_app.1.x6zhcejkwzz4@satoshis-games    | postgres:5432 - accepting connections
chat_app.1.x6zhcejkwzz4@satoshis-games    | + echo 'Database ready to accept connections.'
chat_app.1.x6zhcejkwzz4@satoshis-games    | + bundle install
chat_app.1.x6zhcejkwzz4@satoshis-games    | Database ready to accept connections.
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rake 13.0.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using concurrent-ruby 1.1.5
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using i18n 1.8.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using minitest 5.14.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using thread_safe 0.3.6
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using tzinfo 1.2.6
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using zeitwerk 2.2.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using activesupport 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using builder 3.2.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using erubi 1.9.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mini_portile2 2.4.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using nokogiri 1.10.8
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rails-dom-testing 2.0.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using crass 1.0.6
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using loofah 2.4.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rails-html-sanitizer 1.3.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using actionview 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rack 2.1.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rack-test 1.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using actionpack 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using nio4r 2.5.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using websocket-extensions 0.1.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using websocket-driver 0.7.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using actioncable 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using globalid 0.4.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using activejob 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using activemodel 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using activerecord 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mimemagic 0.3.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using marcel 0.3.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using activestorage 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mini_mime 1.0.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mail 2.7.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using actionmailbox 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using actionmailer 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using actiontext 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using acts-as-taggable-on 6.5.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using public_suffix 4.0.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using addressable 2.7.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using ast 2.4.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using attr_extras 6.2.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using aws-eventstream 1.0.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using aws-partitions 1.269.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using aws-sigv4 1.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using jmespath 1.4.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using aws-sdk-core 3.89.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using aws-sdk-kms 1.28.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using aws-sdk-s3 1.60.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using descendants_tracker 0.0.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using ice_nine 0.11.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using axiom-types 0.1.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using multipart-post 2.1.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using faraday 0.17.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using faraday_middleware 0.14.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using azure-core 0.1.15
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using azure-storage 0.15.0.preview
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using bcrypt 3.1.13
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using msgpack 1.3.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using bootsnap 1.4.5
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using brakeman 4.7.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using browser 3.0.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using bundler 2.1.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using json_pure 2.2.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using http-accept 1.7.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using unf_ext 0.0.7.6
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using unf 0.1.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using domain_name 0.5.20190701
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using http-cookie 1.0.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mime-types-data 3.2019.1009
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mime-types 3.3.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using netrc 0.11.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rest-client 2.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using chargebee 2.7.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using coercible 1.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using connection_pool 2.2.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using declarative 0.0.10
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using declarative-option 0.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using orm_adapter 0.5.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using method_source 0.9.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using thor 0.20.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using railties 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using responders 3.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using warden 1.2.8
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using devise 4.7.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using sprockets 4.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using sprockets-rails 3.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rails 6.0.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using devise_token_auth 1.1.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using digest-crc 0.4.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using dotenv 2.7.5
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using dotenv-rails 2.7.5
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using equalizer 0.0.11
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using execjs 2.7.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using multi_xml 0.6.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using httparty 0.17.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using facebook-messenger 1.4.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using flag_shih_tzu 0.3.23
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using foreman 0.87.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using jwt 2.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using memoist 0.16.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using multi_json 1.14.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using os 1.0.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using signet 0.12.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using googleauth 0.10.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using httpclient 2.8.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using uber 0.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using representable 3.0.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using retriable 3.1.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using google-api-client 0.36.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using google-cloud-env 1.3.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using google-cloud-errors 1.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using google-cloud-core 1.5.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using google-cloud-storage 1.25.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using haikunator 1.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using hashie 4.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using inflecto 0.0.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using jbuilder 2.9.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using json 2.3.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using kaminari-core 1.2.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using kaminari-actionview 1.2.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using kaminari-activerecord 1.2.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using kaminari 1.2.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using koala 3.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using mini_magick 4.10.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using nightfury 1.0.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using oauth 0.5.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using parser 2.7.0.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using pg 1.2.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using puma 4.3.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using pundit 2.1.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rack-cache 1.11.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rack-cors 1.1.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rack-protection 2.0.8.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using rack-proxy 0.6.5
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using redis 4.1.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using redis-namespace 1.7.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using redis-store 1.8.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using redis-rack-cache 2.2.1
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using scout_apm 2.6.6
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using sentry-raven 2.13.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using sidekiq 6.0.4
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using virtus 1.0.5
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using telegram-bot-ruby 0.12.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using time_diff 0.3.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using twitty 0.1.0 from https://github.com/chatwoot/twitty (at master@c1edd55)
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using tzinfo-data 1.2019.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using uglifier 4.2.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using valid_email2 3.1.3
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using webpacker 4.2.2
chat_app.1.x6zhcejkwzz4@satoshis-games    | Using wisper 2.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | Bundle complete! 68 Gemfile dependencies, 150 gems now installed.
chat_app.1.x6zhcejkwzz4@satoshis-games    | Gems in the groups development and test were not installed.
chat_app.1.x6zhcejkwzz4@satoshis-games    | Bundled gems are installed into `/gems`
chat_app.1.x6zhcejkwzz4@satoshis-games    | + BUNDLE='bundle check'
chat_app.1.x6zhcejkwzz4@satoshis-games    | + bundle check
chat_app.1.x6zhcejkwzz4@satoshis-games    | The Gemfile's dependencies are satisfied
chat_app.1.x6zhcejkwzz4@satoshis-games    | + exec bundle exec rails s -p 3000 -b 0.0.0.0
chat_app.1.x6zhcejkwzz4@satoshis-games    | => Booting Puma
chat_app.1.x6zhcejkwzz4@satoshis-games    | => Rails 6.0.2.1 application starting in production 
chat_app.1.x6zhcejkwzz4@satoshis-games    | => Run `rails server --help` for more startup options
chat_app.1.x6zhcejkwzz4@satoshis-games    | Puma starting in single mode...
chat_app.1.x6zhcejkwzz4@satoshis-games    | * Version 4.3.3 (ruby 2.6.5-p114), codename: Mysterious Traveller
chat_app.1.x6zhcejkwzz4@satoshis-games    | * Min threads: 5, max threads: 5
chat_app.1.x6zhcejkwzz4@satoshis-games    | * Environment: production
chat_app.1.x6zhcejkwzz4@satoshis-games    | * Listening on tcp://0.0.0.0:3000
chat_app.1.x6zhcejkwzz4@satoshis-games    | Use Ctrl-C to stop
```
> If you're on heroku, would be worth double checking the redis is provisioned

@tabrza This a soccer swarm stack available at #625 

IMO, the problem is that wen the service is up and running it does not connect to the `db`, I have to manually do either a `bundle exec rails db:reset` or ``bundle exec rails db:setup` to manually seed the database with the default data... Then it works, but importing the previous database results in errors when importing due to Primary Keys in tables.

Then, my solution is dropping the database, creating it again and importing my backup. But because it does not trigger the previous command for `db:reset` or `db:setup` it does not connect to the backend

### SIDE NOTE
This was caused because I was trying to update to the latest version innocently thinking it was stable, and then It all crashed as commented in #699

**Comment by whiteyhat** [comment]
It had to be a compatibility data issues between database versions, by solving #699 on the latest version everything worked fine
